docs/download.md

```diff
@@ -32,7 +32,7 @@ The plugin is tested on the following architectures:
 	NVIDIA Driver*: R470+
 
 	Runtime: 
-		Scala 2.12
+		Scala 2.12, 2.13
```

Comment:
Should remove L40? `Please be aware that we do not currently support Spark builds with Scala 2.13.`

Reply:
Done!